File: CompSci 1/Unit 17, Nested Loops/Patterns/Patterns.java
import java.util.Scanner;

public class Patterns {
    public static void main(String[] args)
    {
        Scanner input = new Scanner(System.in);

        System.out.print("Enter a number: ");
        int num = input.nextInt();

        for(int i = 0; i < num; i++)
        {
            for(int j = 0; j < i; j++)
            {
                System.out.print("*");
            }
            System.out.println();
        }
        for(int i = num; i > 0; i--)
        {
            for(int j = 0; j < i; j++)
            {
                System.out.print("*");
            }
            System.out.println();
        }
    }
}

File: CompSci 1/Unit 17, Nested Loops/Patterns/Patterns.kt
import java.util.Scanner;

fun main(args: Array<String>) {
    val input = Scanner(System.`in`)

    print("Enter a number: ")
    val num = input.nextInt()

    for (i in 0 until num) {
        for (j in 0 until i) {
            print("*")
        }
        println()
    }
    for (i in num downTo 1) {
        for (j in 0 until i) {
            print("*")
        }
        println()
    }
}